pkg/pool-weighted/contracts/WeightedMath.sol

```diff
@@ -444,15 +443,12 @@ library WeightedMath {
         //
         // Compute the portion of the invariant increase due to fees
         uint256 supplyGrowthRatio = postJoinExitSupply.divDown(preJoinExitSupply);
-        uint256 feelessInvariant = preJoinExitInvariant.mulDown(supplyGrowthRatio);
 
-        uint256 invariantDeltaFromFees = postJoinExitInvariant - feelessInvariant;
+        uint256 swapFeesPercentage = FixedPoint.ONE - supplyGrowthRatio.divDown(invariantRatio);
 
         // To convert to a percentage of pool ownership, multiply by the rate,
         // then normalize against the final invariant
```

Comment:
This comment is a bit stale now

Reply:
Yep, and the one above as well. Working on it now